client/me/purchases/membership-item/index.tsx

```diff
@@ -40,6 +40,7 @@ const MembershipTerms = ( { subscription }: { subscription: MembershipSubscripti
 
 const SiteLink = ( { subscription }: { subscription: MembershipSubscription } ) => {
 	const translate = useTranslate();
+	const siteUrl = subscription.site_url.replace( /^(https?:|)\/\//, '' );
```

Comment:
Just out of curiosity, what's the vertical bar for here?

Reply:
cc @michaeldcain (he helped me write that :D)

Reply:
To my eye, it reads as "replace a leading 'http' followed by an optional 's' _or nothing_ with nothing" and I wonder if maybe it was supposed to be:

```suggestion
	const siteUrl = subscription.site_url.replace( /^https?:\/\//, '' );
```

But it's also possible that I don't know as much as I think I do, which is why I wanted to check!

Reply:
Payton's is better. 🙄

Reply:
ahah ok. I can switch it.

Reply:
Tested and confirmed it still works :D